app/views/admin/campaigns/index.html.haml

```diff
@@ -18,16 +18,16 @@
       %tbody
         - @campaigns.each do |campaign|
           %tr
-            %td{'id'=> "name_#{campaign.id}"}
+            %td{ 'id' => "name_#{campaign.id}" }
```

Comment:
Let's leave it as it is for now.
In another PR we can enable haml-lint and start adding the rules one by one in different PRs, so that the changes are not so huge.